lib/msal-common/src/telemetry/performance/IPerformanceClient.ts

```diff
@@ -31,6 +32,7 @@ export interface IPerformanceClient {
         fields: { [key: string]: {} | undefined },
         correlationId: string
     ): void;
+    addError(error: Error, correlationId: string, maxStackSize: number): void;
```

Comment:
Adding functions to this interface is potentially a breaking change - is there a way to add this functionality using the existing methods?

Reply:
Done. Looks a bit ugly but does the job with regards to perf interface immutability.

Reply:
Thanks this looks good